ir/src/transition_constraints/graph.rs

```diff
@@ -165,16 +176,13 @@ impl AlgebraicGraph {
         }
     }
 
-    fn insert_variable(
+    fn insert_elem(
```

Comment:
Should this maybe be `insert_symbol()` or maybe `insert_symbol_access()` to be more consistent with `insert_vector_access()` and `insert_matrix_access()`?

Reply:
Changed it to `insert_symbol_access()`.